### ISSUE TITLE
new node type FloatingWeir (i.e. Tabulated QdH-relation)

### ISSUE THREAD

**Issue by gijsber** [Deltares/Ribasim#753]
During our LHM-schemtisation workshop we discussed how we could ensure that simple situations do not need complex model setups.

E.g. we have a polder system with an inlet from the boezem, 4 cascading polders and an pumpstation.
We want to preserve level control on all polders. In reality, when the water level drops, each weir will open a bit more to let more water in.
This could be modelled by a series of 'floating weirs', i.e. a ratingcurve which computed Q based on dH. 
When the water level of the downstream basin drops, dH increases and Q rises to compensate the water level drop.

**Comment by gijsber** [comment]
Correct. Finding the right name might be a challenge, TabulatedQdeltaH seems acceptabel to me.

**Comment by gijsber** [comment]
to be agreed/confirmed by @visr . 
From a data structure perspective it could be a copy of TabulatedratingCurve with a deltaH instead of an H.
Q is indeed a function of the level difference before and after the node

**Comment by visr** [comment]
I don't think that the suggested new node type would solve the use case sketched in the first post. In that use case I think having $\Delta h$ would be one of the things we need, but the absolute values are also still needed.

We should let this and #719 soak in a bit, and see if there is a more general solution that would help use cases like this. #720 and being able to take a difference between two things that control listens to ($\Delta h$) might help there as well.

Of course a general solution may not necessarily be easy to use, so adding node types for this is also an option.

**Comment by deltamarnix** [comment]
@SnippenE @visr Could you two discuss a proposal?
